docs/source/release_notes/release-notes.rst

```diff
@@ -41,6 +41,10 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed|
+         - Multinode Cassandra: ``getAllTimestamps`` and ``getRangeOfTimestamps`` now work as long as quorum for reading is achieved.
```

Comment:
How about "`getAllTimestamps` and `getRangeOfTimestamps` now work on a multinode Cassandra cluster with one down node"?